docs/bin/plugin_formatter.py

```diff
@@ -132,6 +133,17 @@ def rst_xline(width, char="="):
     return char * width
 
 
+def to_boolean(value):
+    ''' return default value in boolean type '''
```

Comment:
Tripple double quotes (`"""`). No spaces around the sentence. Period in the end of the sentense.

Reply:
fixed!